src/biogeophys/TemperatureType.F90

```diff
@@ -410,6 +412,16 @@ subroutine InitHistory(this, bounds, is_simple_buildtemp, is_prog_buildtemp )
          avgflag='A', long_name='soil temperature in top 10cm of soil', &
          ptr_col=this%t_soi10cm_col, set_urb=spval)
 
+    this%t_sno_mul_mss_col(begc:endc) = spval
+    call hist_addfld1d (fname='SNOTXMASS',  units='K kg/m2', &
+         avgflag='A', long_name='snow temperature times layer mass, layer sum', &
```

Comment:
This and the related _ICE variable should probably add in the long_name something like this:

```Fortran
         avgflag='A', &
         long_name='snow temperature times layer mass, layer sum; '// &
         'to get mass-weighted temperature, divide by (SNOWICE+SNOWLIQ)', &
```

Reply:
Great suggestion and I agree with your wording. Thanks